app/src/main/kotlin/compiler/syntax/LanguageTokens.kt

```diff
@@ -30,6 +30,8 @@ object LanguageTokens {
             Pair(TokenType.RETURN, "zwróć"),
             Pair(TokenType.RETURN_UNIT, "zakończ"),
             Pair(TokenType.FUNCTION, "czynność"),
+            Pair(TokenType.FOREIGN, "obca"),
```

Comment:
Ja bym nazwał `zewnętrzna` (od `extern`).

Reply:
I was considering this name too. I choose "obca" because that one would make one of our example programs invalid.

Reply:
But I agree that "zewnętrzna" sounds better, so I'll change the example program.